test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,93 @@
+require "test_helper"
+
+describe WorksController do
+  before do
+    @work = works(:book1)
+  end
+  
+  describe "index" do
+    it "responds with success when there are works saved" do
+
+      get(works_path)
+     
+      must_respond_with :success
+    end
+
+    it "responds with success when there are no works saved" do
+     
+      Work.delete_all
+
+      get(works_path)
+     
+      must_respond_with :success
+    end
+  end
+
+  describe "show" do
+    it "responds with success when showing an existing valid work" do
+
+      get(work_path(@work.id))
+      must_respond_with :success
+    end
+
+    it "responds with 404 with an invalid work id" do
+      get(work_path(00000000))
```

Comment:
In Ruby `00000000` is the same as `0`.

```suggestion
      get(work_path(0))
```